Stylings for the new header and footer

i0sea opened the pull request:
Updated stylings for header and footer. Some styles didn't port over and were made manually. This does not include the search bar nor mobile menu.

i0sea committed:
new nav

i0sea committed:
updates

i0sea committed:
remove search

i0sea committed:
force styling of buttons

thecristen reviewed (changes requested):
Thanks @i0sea. Can you remove the `/node_modules/.bin` folder that ended up in here?
Additionally, you've included multiple copies of the JS and CSS files here -- likely from running the mix task multiple times. Please delete all of the extras and keep the style linked from the `head.html` and the JavaScript linked from the `scripts.html`

i0sea committed:
remove node_modules

i0sea committed:
rm extra css and js files

i0sea committed:
remove excess

thecristen commented on header.html:
```diff
@@ -30,790 +44,1579 @@
+        </div>
+      </nav>
+  </div>
+  <nav class="m-menu--mobile" id="navmenu">
```
Let's either try to make the mobile menu work (when clicking the "menu" button) or delete this menu entirely. Leaning toward deleting this nav

thecristen commented on header.html:
```diff
@@ -30,790 +44,1579 @@
-                <span class="nav-link-name">
-                  Contact Us
+        </div>
+        <div class="m-menu__language">
```
Ah interesting, I wouldn't expect this div to still be here

thecristen reviewed (changes requested):
Thanks!!

There's a few minor style issues. Maybe it's worth seeing if we can import anything else into to the CSS file to fix it. See the spacing in the callout sections of the menu:

<img width="1306" alt="image" src="https://user-images.githubusercontent.com/2136286/162064134-38fb04ac-df7c-43ba-8027-7d07b5859b4c.png">

I don't know how we pulled it off but there's an extra menu button on smaller screens!

<img width="643" alt="image" src="https://user-images.githubusercontent.com/2136286/162064308-3d880df1-3724-46ec-9644-48b159ba653d.png">

I'd like to keep the first Menu button (which happens to link to the /menu page) and remove the second one (which doesn't do anything)

i0sea commented:
Curious why the spacing on the contact page and between the menu buttons was off even though those styles should've been imported from `variables.scss`. 🤔

i0sea committed:
rm extra small screen menu

i0sea commented:
@thecristen  Delete all mobile menu buttons or just keep the one that links to the js menu, given that the js mobile menu isn't rendering?

thecristen commented:
Keep the one linking to the JS menu!

i0sea commented:
Removed extra mobile menu button, kept the functioning one. Imported core, app, and changed variables to vairiables.scss but the contact submenu remains the same 😖
